scripts/subset_crams.py

```diff
@@ -80,15 +83,16 @@ def main(beds: str):
         cpg_id = sample_map[ext_id]
         cram_file = cram_map[cpg_id]
         cram_job = batch.new_job(name=f'subset CRAM {ext_id}/{cpg_id}')
+
+        # authenticate credentials in job
+        authenticate_cloud_credentials_in_job(cram_job)
+
+        # run the job inside the samtools image
         cram_job.image(samtools_image)
 
         # big storage, probably not huge on compute (small regions)
         cram_job.storage('60G')
```

Comment:
This really just needs to be enough to fit the reference, right?

Reply:
Reference and any temp/intermediate files the threading would cause - I'll reduce to much lower than this